src/pages/app-development/app-submission-guidelines.md

```diff
@@ -0,0 +1,182 @@
+
+## Review process
+
+When you submit your Commerce App Builder extension, our review team will evaluate your submission against the criteria outlined on this page.
+
+- **[Requirements](#requirements)** - Critical requirements that must be addressed before approval
+- **[Best practices](#best-practices)** - Recommended improvements that enhance quality but don't block approval
+
+## If your app is rejected
+
+If your submission is rejected:
+
+1. Review the specific feedback provided by the review team.
+1. Address all requirements listed in the rejection notice.
+1. Consider implementing best practices to improve quality.
+1. Resubmit your updated extension.
+
+## Requirements
+
+The following requirements must be met for your app to be accepted. If your app does not meet these requirements, it will be rejected.
+
+### Documentation
+
+- Installation guide clarity
+  - **Prerequisites**: Add clear prerequisites (example: Admin UI SDK must be enabled).
+  - **Version requirements**: Indicate the required module versions (example: Admin UI SDK minimum version `3.0.0`).
+  - **Developer documentation**: Include links to relevant Adobe developer documentation (example: App Builder [getting started guide](https://developer.adobe.com/app-builder/docs/get_started/)).
+  - **Environment setup**: Provide a `.env.dist` file with all required environment variables, clearly labeled with guidance.
+  - **PaaS support**: Documentation must include installation steps for PaaS merchants. This means if you are submitting an app for Adobe Commerce as a Cloud Service (SaaS), you also need to support Adobe Commerce on cloud infrastructure (PaaS) and Adobe Commerce on-premises (on-prem).
+
+- Security awareness
+  - **Screenshot security**: Ensure no screenshots contain access tokens, secrets, or API keys.
+  - **Sensitive data**: Do not expose credentials or sensitive information in the documentation.
+
+- `README` content quality
+  - **Project-specific language**: Use project-specific language. Avoid generic references to "Adobe" or "starter kit".
+  - **Configuration details**: Clearly list the app configuration and any third-party service-specific setup steps.
+  - **Usage instructions**: Add instructions on how to use the app post-installation.
+  - **Professional writing**: Remove any placeholder content and fix any typos.
+
+### Security
+
+- Authentication and authorization
+  - **Admin action security**: All admin-supporting actions must use `require-adobe-auth: true` in the action configuration.
+    - This requires that PaaS apps use Admin UI SDK 3.0 or later.
+  - **Webhook protection**: For webhooks, actions need to be protected by [signature verification](https://developer.adobe.com/commerce/extensibility/webhooks/signature-verification/).
+
+    ```yaml
+    inputs:
+    COMMERCE_WEBHOOKS_PUBLIC_KEY: $COMMERCE_WEBHOOKS_PUBLIC_KEY
+    annotations:
+    require-adobe-auth: false
+    raw-http: true
+    ```
```

Comment:
Indentation is required for YAML, also removed `require-adobe-auth` as it's an independent config that doesn't need to be always turn off.
```suggestion
    ```yaml
    inputs:
        COMMERCE_WEBHOOKS_PUBLIC_KEY: $COMMERCE_WEBHOOKS_PUBLIC_KEY
    annotations:
        raw-http: true
    ```
```